ntbs-ui-tests/start_selenium.ps1

```diff
@@ -1,5 +1,17 @@
 #!/usr/local/bin/powershell
 
-.\node_modules\.bin\selenium-standalone.ps1 install
+ping 127.0.0.1
```

Comment:
Why are we pinging ourselves here?

Reply:
Ah - this (and also the `netstat` command below) were added when I was trying to debug why it wasn't working on a windows server. I need to remove these.